lib/LedgerSMB/File.pm

```diff
@@ -247,9 +247,12 @@ sub get_for_template{
     warn 'entering get_for_template';
 
     my @results = $self->call_procedure(
-                 funcname => 'file__get_for_template',
-                      args => [$args->{ref_key}, $args->{file_class}]
-     );
+        funcname => 'file__get_for_template',
+        args => [
+            $args->{ref_key},
+            $args->{file_class},
+        ],
+    );
     if ( -d $LedgerSMB::Sysconfig::tempdir . '/' . $$){
         die 'directory exists';
```

Comment:
I'd rather refactor this and the associated cleanup to use File::Temp, but for now will expand the error.